provider/vsphere/client.go

```diff
@@ -200,12 +243,23 @@ func (c *client) Refresh(v *mo.VirtualMachine) error {
 }
 
 func (c *client) getVm(name string) (*mo.VirtualMachine, error) {
```

Comment:
Well this is out of the scope of the present change but since its a free rename i did it.